tests/PhpseclibFunctionalTestCase.php

```diff
@@ -44,7 +44,7 @@ public static function setUpBeforeClass()
     protected function requireEnv($variable, $message = null)
     {
         if ($this->_getEnv($variable) === false) {
-            $msg = $message ? $message : sprintf(
+            $msg = $message ?: sprintf(
```

Comment:
> We can try targeting 1.0 for this PR - hopefully it will be no conflicts.

Well, for 1.0, you'd need to remove the `dirname(__FILE__)` replacements. What it was replaced with - ` __DIR__` - only works on PHP 5.3 and later. The 1.0 branch is supposed to be compatible with PHP 4.x (assuming the use of PHP_Compat).

I'd say target some select things for the 1.0 branch and those things that can't be utilized in the 1.0 branch submit a separate PR for after the changes for the 1.0 branch have been merged.